extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -354,18 +354,22 @@ require(['wikia.window', 'jquery', 'wikia.nirvana', 'wikia.tracker', 'JSMessages
 								if (errReason === 'imageMissing') {
 									return imageMissingError;
 								}
+								if (errReason === 'duplicatedLabel') {
+									return duplicateError;
+								}
 								return errReason;
 							}
 
 							if (data.error) {
-								var items = $form.find('.item-input, .section-input');
+								var $items = $form.find('.item-input'),
+									$sections = $form.find('.section-input');
 
-								$.each(data.error, function() {
-									var errTitle = this.title,
-										errReason = this.reason,
+								[].forEach.call(data.error, function(err) {
```

Comment:
Would it be difficult to extract this function? There is sooo much indentation here.